spec/requests/posts_spec.rb

```diff
@@ -1,35 +1,41 @@
 require 'rails_helper'
 
 describe 'Posts', type: :request do
-  describe 'GET #index' do
-    before(:example) { get '/users/1/posts' }
+  before(:each) do
+    user = User.create!(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
```

Comment:
Good work implementing some tests! 

- [ ] Kindly fix the error in the terminal


![image](https://github.com/Roland-Ntwali/Blog-App/assets/93630700/53e62fb1-1044-440e-940a-bbb14b9107c2)